cds-snc/covid-alert-app
# Change exposureStatusType to an enum.

# Summary

This is a code optimization change. This changes the literal strings for exposureStatusType to an enum, so there are no magic strings. The functionality of the app should not be altered by this change. The goal is to make the code more maintainable.

# Test instructions

To test this change, run the app, and change the exposureStatus of the device from 'monitoring', to 'exposed' (by having it sit near a diagnosed phone for 15+ minutes. The exposureStatus should be changed and maintained between app launches.
